src/drupal8/docs/customise/commands.md

```diff
@@ -0,0 +1,42 @@
+# Adding custom commands
+
+Adding commands can help with productivity and standardise tasks.  
+Information on writing commands can be found [here](https://github.com/my127/workspace/blob/0.1.x/docs/types/command.md)
+
+Here are some example that might be a good starting point.
+
+
+### Refresh script - `ws drupal-refresh`
+```yaml
+command('drupal-refresh'):
+  exec: |
+    #!bash(workspace:/)|@
+    ws composer install
+    ws drush -y cr
+    ws drush -y updb
+    ws frontend install
+```
+
+
+### Database import - `ws import-db path/to/db.sql`
+```yaml
+command('drush-import-db <alias> <filename>'):
+  exec: |
+    #!bash|=
+    database_filename="={ input.argument('filename') }"
+    alias="={ input.argument('alias') }"
+    ws drush $alias "sqlc < $database_filename"
+```
+
+
+### Frontend development - `ws frontend-start`
+```yaml
+attribute('frontend.path'): "/app/docroot/themes/custom/mytheme"
+
+# ...
+
+command('frontend-start'):
```

Comment:
I think this could be adapted to be `ws frontend watch` https://github.com/inviqa/harness-base-php/blob/0.6.x/src/_base/harness/config/commands.yml#L135-L140

Need to set `attribute('frontend.watch'): 'yarn dev'` and it will work with the frontend.path attribute above

Reply:
Thanks, I've removed this to avoid confusion with the commands that are already defined.

Reply:
Ace!